src/test/java/com/uid2/admin/vertx/ClientKeyServiceTest.java

```diff
@@ -80,13 +73,8 @@ public void clientRenameWithExistingName(Vertx vertx, VertxTestContext testConte
                 new LegacyClientBuilder().withName("test_client1").build()
         );
 
-        post(vertx, "api/client/rename?oldName=test_client&newName=test_client1", "", ar -> {
-            HttpResponse<Buffer> response = ar.result();
-            assertAll(
-                    "clientRenameWithExistingName",
-                    () -> assertTrue(ar.succeeded()),
-                    () -> assertEquals(400, response.statusCode())
-            );
+        post(vertx, testContext, "api/client/rename?oldName=test_client&newName=test_client1", "", response -> {
```

Comment:
nitpick: Could use `expectHttpError(400)`

Reply:
Good point, I've updated that